website/content/en/docs/contribution-guidelines/local-changes.md

```diff
@@ -1,90 +0,0 @@
----
-title: Testing Changes Locally
-linkTitle: Local Changes
```

Comment:
We are removing this info. However, it is very valid for new contributors and new team members. I already need to share it with more than one person. Where it will be addressed?

Reply:
These test descriptions are out of date and are difficult to keep up to date because our tests change a lot with additions and bug fixes.